--- pristine/build/org.sthaid.entropy/app/jni/src/Android.mk	2020-11-28 10:14:10.422944019 -0500
+++ SDL2-2.0.12/build/org.sthaid.entropy/app/jni/src/Android.mk	2020-11-28 10:17:35.974934805 -0500
@@ -5,14 +5,16 @@
 LOCAL_MODULE := main
 
 SDL_PATH := ../SDL
+SDL_TTF_PATH := ../SDL2_ttf-2.0.15
 
-LOCAL_C_INCLUDES := $(LOCAL_PATH)/$(SDL_PATH)/include
+LOCAL_C_INCLUDES := $(LOCAL_PATH)/$(SDL_PATH)/include \
+                    $(LOCAL_PATH)/$(SDL_TTF_PATH) 
 
 # Add your application source files here...
-LOCAL_SRC_FILES :=  stub.c
+LOCAL_SRC_FILES :=  main.c sim_container.c sim_gravity.c sim_randomwalk.c sim_universe.c util.c version.c
 
-LOCAL_SHARED_LIBRARIES := SDL2
+LOCAL_SHARED_LIBRARIES := SDL2 SDL2_ttf
 
-LOCAL_LDLIBS := -lGLESv1_CM -lGLESv2 -llog
+LOCAL_LDLIBS := -lGLESv1_CM -lGLESv2 -llog -landroid
 
 include $(BUILD_SHARED_LIBRARY)
--- pristine/build/org.sthaid.entropy/app/src/main/res/values/strings.xml	2020-09-14 19:51:27.644129724 -0400
+++ SDL2-2.0.12/build/org.sthaid.entropy/app/src/main/res/values/strings.xml	2020-09-13 19:38:52.284118088 -0400
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">Game</string>
+    <string name="app_name">Entropy</string>
 </resources>
--- pristine/build/org.sthaid.entropy/app/src/main/AndroidManifest.xml	2020-11-29 07:16:29.971685670 -0500
+++ SDL2-2.0.12/build/org.sthaid.entropy/app/src/main/AndroidManifest.xml	2020-11-29 07:19:02.919111518 -0500
@@ -4,8 +4,8 @@
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.sthaid.entropy"
-    android:versionCode="1"
-    android:versionName="1.0"
+    android:versionCode="1"
+    android:versionName="1.0"
     android:installLocation="auto">
 
     <!-- OpenGL ES 2.0 -->
@@ -71,6 +71,7 @@
             android:alwaysRetainTaskState="true"
             android:launchMode="singleInstance"
             android:configChanges="orientation|uiMode|screenLayout|screenSize|smallestScreenSize|keyboard|keyboardHidden|navigation"
+            android:screenOrientation="landscape"
             >
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
--- pristine/build/org.sthaid.entropy/app/build.gradle	2020-11-29 07:32:11.772992512 -0500
+++ SDL2-2.0.12/build/org.sthaid.entropy/app/build.gradle	2020-12-01 08:13:43.970066344 -0500
@@ -8,15 +8,15 @@
 }
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 29
     defaultConfig {
         if (buildAsApplication) {
             applicationId "org.sthaid.entropy"
         }
         minSdkVersion 16
-        targetSdkVersion 26
-        versionCode 1
-        versionName "1.0"
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
         externalNativeBuild {
             ndkBuild {
                 arguments "APP_PLATFORM=android-16"
